src/draw/nema_gfx/lv_draw_nema_gfx_label.c

```diff
@@ -95,6 +112,15 @@ void lv_draw_nema_gfx_label(lv_draw_unit_t * draw_unit, const lv_draw_label_dsc_
 
     nema_set_clip(clip_area.x1, clip_area.y1, lv_area_get_width(&clip_area), lv_area_get_height(&clip_area));
 
+#if LV_USE_FREETYPE && LV_USE_NEMA_VG
+    /*We need to initiallize in every new label task*/
+    static bool is_inited = false;
```

Comment:
You can refer to this PR: https://github.com/lvgl/lvgl/pull/7363 to make modifications without using the `is_inited` variable to implement delayed initialization.